week-3/slideshow/index.html

```diff
@@ -3,12 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Image Carousel</title>
     <script defer src="slideshow.js"></script>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
-    <button type="button" id="backward-btn">Backwards</button>
-    <button type="button" id="forward-btn">Forward</button>
+    <section class="box">
+      <button type="button" id="auto-backward-btn">Auto Backwards</button>
+      <button type="button" id="backward-btn">Backwards</button>
+      <button type="button" id="stop-btn">Stop</button>
+      <button type="button" id="forward-btn">Forward</button>
+      <button type="button" id="auto-forward-btn">Auto Forward</button>
+    </section>
```

Comment:
The code is clean, concise and readable. The unique ID's you have given to buttons is thoughful and can simplify the development process and make the code more maintainable in the future.